src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Class implements Value Object.
+ */
+public class LoyaltyCard {
+    /**
+     * Customer's id.
```

Comment:
No, the whole card is the ID of a customer. It's just like passport for a citizen in the context of a country, but simpler, and in the context of a shop.